packages/tabs/test/overflow.test.js

```diff
@@ -0,0 +1,142 @@
+import { expect } from '@vaadin/chai-plugins';
+import { fixtureSync, listenOnce, nextRender, nextResize } from '@vaadin/testing-helpers';
+import './tabs-test-styles.js';
+import '../src/vaadin-tabs.js';
+
+describe('overflow', () => {
+  let tabs;
+
+  beforeEach(async () => {
+    tabs = fixtureSync(`
+      <vaadin-tabs>
+        <vaadin-tab>Foo</vaadin-tab>
+        <vaadin-tab>Bar</vaadin-tab>
+        <vaadin-tab>Some</vaadin-tab>
+        <span></span>
+        <vaadin-tab disabled>Baz</vaadin-tab>
+        <vaadin-tab>
+          <a>Baz</a>
+        </vaadin-tab>
+      </vaadin-tabs>
+    `);
+    await nextRender();
+    tabs._observer.flush();
+  });
+
+  ['horizontal', 'vertical'].forEach((orientation) => {
+    beforeEach(() => {
```

Comment:
You need to move this to the `describe` below.

Reply:
Thanks, fixed.